src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskAssignByIdControllerCFTTest.java

```diff
@@ -123,6 +124,7 @@ public void should_return_a_204_when_assigning_a_task_by_id_with_restricted_role
 
         assertions.taskVariableWasUpdated(taskVariables.getProcessInstanceId(), "taskState", "assigned");
         assertions.taskStateWasUpdatedinDatabase(taskId, "assigned", authenticationHeaders);
+        assertions.taskFieldWasUpdatedInDatabase(taskId, "assignee",assigneeId, authenticationHeaders);
```

Comment:
Nice. Could we add a similar check for the claim CFT-FT? Possible in this ticket?

Reply:
These will be addressed in this ticket
https://tools.hmcts.net/jira/browse/RWA-919